Client/src/main/java/com/lnc/util/ToJsonConversion.java

```diff
@@ -0,0 +1,129 @@
+package com.lnc.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.lnc.model.*;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ToJsonConversion {
```

Comment:
class name is not really not aligned with purpose of the class, it not for just JSON conversion, it is setting up the APIs